src/bridging/types.ts

```diff
@@ -0,0 +1,174 @@
+import { latest as latestAppData } from '@cowprotocol/app-data'
+import { ethers } from 'ethers'
+import { ChainInfo, ChainId } from '../common'
+import { TokenInfo } from '../common/tokens'
+import { Address } from '../order-book'
+import { EvmCall } from '../common/transaction'
+
+export interface BridgeProviderInfo {
+  name: string
+  logoUrl: string
+}
+
+export interface WithSellToken {
+  sellTokenChainId: ChainId
+  sellTokenAddress: Address
+  sellTokenDecimals: number
+}
+
+export interface WithBuyToken {
+  buyTokenChainId: ChainId
+  buyTokenAddress: Address
+  buyTokenDecimals: number
+}
+
+export interface GetBuyTokensParams extends Partial<WithSellToken> {
+  targetChainId: ChainId
+}
+
+/**
+ * Parameters for getting a bridge quote
+ */
+export interface QuoteBridgeRequest extends WithSellToken, WithBuyToken {
+  type: 'SELL' // We make it explicit that only SELL is supported for now
```

Comment:
Why is that?

Reply:
Because buy orders are harder....

How do you ensure you are able to buy exactly an amount on the other side of the bridge?

Reply:
This is to limit scope

Reply:
Ahhhhh right, that's for the bridge!
I was thinking about the SWAP.
Disregard.